wqflask/wqflask/templates/facilities.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block title %}Facilities{% endblock %}
+
+{% block css %}
+<!-- <link rel="stylesheet" type="text/css" href="/static/new/css/markdown.css" /> -->
+{% endblock %}
```

Comment:
**suggestion:nitpick:** You should probably just remove this.

Any reason why it's commented out though?